examples/server_certificate.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"os"
+)
+
+func main() {
+	var (
+		ClientOV                *ov.OVClient
+		server_certificate_ip                 = "172.18.13.11"
+		server_certificate_name               = "new_test_certificate"
+		new_cert_base64data     utils.Nstring = "---BEGIN CERTIFICATE----END CERTIFICATE------"
+	)
+	ovc := ClientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		800,
+		"")
+	ovVer, _ := ovc.GetAPIVersion()
+	fmt.Println(ovVer)
+
+	server_cert, err := ovc.GetServerCertificateByIp(server_certificate_ip)
+	if err != nil {
+		fmt.Println(err)
+	} else {
+		fmt.Println(server_cert)
+	}
+	server_cert.CertificateDetails[0].AliasName = server_certificate_name
+	fmt.Println(server_cert.CertificateDetails[0].AliasName)
+
+	er := ovc.CreateServerCertificate(server_cert)
+	if er != nil {
+		fmt.Println("............... Adding Server Certificate Failed:", er)
+	} else {
+		fmt.Println(".... Adding Server Certificate Success")
+	}
+	fmt.Println("#................... Server Certificate by Name ...............#")
+	server_certn, err := ovc.GetServerCertificateByName(server_certificate_name)
+	if err != nil {
+		fmt.Println(err)
+	} else {
+		fmt.Println(server_certn)
+	}
+
+	server_certi, _ := ovc.GetServerCertificateByName(server_certificate_name)
```

Comment:
again avoid calling same calls when you can reuse the server_certn for the update

Reply:
Done